pkg/expressions/json_templates.go

```diff
@@ -1,17 +1,31 @@
 package expressions
 
 import (
-	"bytes"
 	"encoding/json"
 	"fmt"
-	"io"
 	"strconv"
 	"strings"
 
 	"github.com/expr-lang/expr"
-	"github.com/valyala/fasttemplate"
 )
 
+const (
+	stdStartDelim = "${{"
+	stdEndDelim   = "}}"
+	altStartDelim = "${%"
+	altEndDelim   = "%}"
+)
+
+type supportedDelimiterSet struct {
+	start string
+	end   string
+}
+
+var supportedDelimiters = []supportedDelimiterSet{
+	{start: stdStartDelim, end: stdEndDelim},
+	{start: altStartDelim, end: altEndDelim},
+}
```

Comment:
1. There's no longer any good reason to have constants for the start/end delimiters.

2. Nit: I would also recommend moving these type defs to just above `nextExpression()`. It's the _only_ place any of this is used. When something is used only in one place, readability is often improved by keeping its definition close to that one place.

```suggestion
type supportedDelimiterSet struct {
	start string
	end   string
}

var supportedDelimiters = []supportedDelimiterSet{
	{start: "${{", end: "}}"},
	{start: "${%", end: "%}"},
}
```